.circleci/config.yml

```diff
@@ -86,6 +86,13 @@ workflows:
           requires:
               - verify-code
           validation: true
+      - notify-private-ci:
```

Comment:
```suggestion
      - sync-internal:
```

Reply:
`sync-internal` feels like we are going to sync `internal` repo rather than `private` one. I guess the current name is fine, or maybe `sync-private` ?. upto @yunikkk to decide 😉

Reply:
@ank27 used `internal` wording as Peng suggested